### PR TITLE
Fix Rubocop violations

### PR DESCRIPTION
**Pull request by spencertransier**
## What does it do?

- This PR fixes some of rubocop violations from `.rubocop_todo.yml`. I went through and removed a section from the todo file, ran Rubocop, then fixed any of the issues it found. 
- It may be helpful to review this PR commit by commit to see which rubocop rule affected which changes. 
- I also regenerated `.rubocop_todo.yml` to get the latest status by running `bundle exec rubocop --regenerate-todo`
- For the [`Naming/VariableName` changes](https://github.com/wordpress-mobile/release-toolkit/commit/be35b01fcebd38c5114780a3a5a6404e9f56187a), I verified that those variables are only used internally by those actions and are not breaking changes for clients. 

## Checklist before requesting a review

- [X] Run `bundle exec rubocop` to test for code style violations and recommendations
- [X] Add Unit Tests (aka `specs/*_spec.rb`) if applicable
- [X] Run `bundle exec rspec` to run the whole test suite and ensure all your tests pass
- [X] Make sure you added an entry in [the `CHANGELOG.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md#trunk) to describe your changes under the appropriate existing `###` subsection of the existing `## Trunk` section.
- [X] If applicable, add an entry in [the `MIGRATION.md` file](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/MIGRATION.md) to describe how the changes will affect the migration from the previous major version and what the clients will need to change and consider.

### CODE REVIEW

**Review by crazytonyli — APPROVED** [review]
I trust Rubocop made the right auto-correct changes. And [the name changes](https://github.com/wordpress-mobile/release-toolkit/commit/be35b01fcebd38c5114780a3a5a6404e9f56187a) looks good to me too.

**Review by iangmaia — APPROVED** [review]
Nice one, @spencertransier 👍 

LGTM. I'll wait for this one to land before I finally merge #489 to avoid conflicts on your side, so that I can also update the code according to the updated rules over there.

**Comment by spencertransier** [comment]
Thanks @iangmaia. Sounds good, I'm excited to have your PR merged!

**Pull request merged**
